tests/target-browser/test_helper.go

```diff
@@ -0,0 +1,210 @@
+	namespace, present := os.LookupEnv(InstallNamespace)
+	if !present {
+		panic("Install Namespace not found in environment")
+	}
+	return namespace
+}
+func GetNFSIPAddr() string {
+	nfsIPAddr, isNFSIPAddrPresent := os.LookupEnv(NFSServerIPAddress)
+	if !isNFSIPAddrPresent {
+		panic("NFS IP address not found in environment.")
+	}
+	return nfsIPAddr
+}
+func GenerateRandomString(n int, isOnlyAlphabetic bool) string {
+	letters := "abcdefghijklmnopqrstuvwxyz"
+	numbers := "1234567890"
+	rand.Seed(time.Now().UnixNano())
+	letterRunes := []rune(letters)
+	if !isOnlyAlphabetic {
+		letterRunes = []rune(letters + numbers)
+	}
+	b := make([]rune, n)
+	for i := range b {
+		randNum, _ := cryptorand.Int(cryptorand.Reader, big.NewInt(int64(len(letterRunes))))
+		b[i] = letterRunes[randNum.Int64()]
+	}
+	return string(b)
+}
+
+func UnMountTarget() {
+	c := fmt.Sprintf("sudo umount %s", targetLocation)
+	command := exec.Command("bash", "-c", c)
+	_, err := command.CombinedOutput()
+	if err != nil {
+		log.Errorf("error %s", err.Error())
+	}
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+}
+
+func makeRandomDirAndMount() {
+
+	var err error
+	randomDirectory = GenerateRandomString(5, false)
+
+	// making directory of random name
+	err = os.MkdirAll(filepath.Join(targetLocation, randomDirectory), 0777)
+	gomega.Expect(err).To(gomega.BeNil())
+	_, err = shell.ChmodR(filepath.Join(targetLocation, randomDirectory), "777")
+	gomega.Expect(err).To(gomega.BeNil())
+	log.Info("directory created:", randomDirectory)
+
+	// unmounting from default target path
+	UnMountTarget()
+	log.Info("unmounted from default path")
+
+	time.Sleep(time.Second * 10)
+
+	// setting "NFS_SERVER_BASE_PATH" to newly formed directory and mounting to it
+	gomega.Expect(os.Setenv(NFSServerBasePath, TargetBrowserDataPath+"/"+randomDirectory)).To(gomega.BeNil())
+	MountTarget()
+	log.Info("mounted to new path")
+	gomega.Expect(err).To(gomega.BeNil())
+
+	time.Sleep(time.Second * 20)
+}
+
+func MountTarget() {
+	targetBrowserPath := os.Getenv(NFSServerBasePath)
+	c := fmt.Sprintf("sudo mount -t nfs -o  nfsvers=4 %s:%s %s", NfsIPAddress, targetBrowserPath, targetLocation)
+	fmt.Printf("Mount command %s\n", c)
+	command := exec.Command("bash", "-c", c)
+	_, err := command.CombinedOutput()
+	if err != nil {
+		log.Errorf("error %s", err.Error())
+	}
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+}
+
+func removeRandomDirAndUnmount() {
+
+	// unmounting from random named directory
+	UnMountTarget()
+	log.Info("unmounted from random named directory", randomDirectory)
+
+	time.Sleep(time.Second * 10)
+
+	// setting "NFS_SERVER_BASE_PATH" to default target path and mounting to it
+	gomega.Expect(os.Setenv(NFSServerBasePath, TargetBrowserDataPath)).To(gomega.BeNil())
+	MountTarget()
+	log.Info("mounted to default path")
+
+	time.Sleep(time.Second * 10)
+
+	// removing random named directory
+	_, _ = shell.RmRf(filepath.Join(targetLocation, randomDirectory))
+
+	log.Info("removed directory")
```

Comment:
If there's err in RmRf cmd, then log statement will be misleading

Reply:
Fixed this